app/controllers/api/v1/reviews_controller.rb

```diff
@@ -6,4 +6,39 @@ def index
   def show
     render json: Review.find(params[:id])
   end
+
+  def create
+
+    @shop = Shop.find(params[:shop_id])
+    @donut = Donut.find(params[:donut_id])
+#    @review = Review.new(review_params)
+
+
+    #@donut.shop = @shop
+    @review = Review.new
+    @review.donut = @donut
+    @review.shop = @shop
+
+    @review.rating = params[:donutReviewRating]
+    @review.body = params[:donutReviewTextBody]
+    @review.price_range = params[:donutReviewPrice]
+
+    if @review.save
+      puts "winner"
```

Comment:
haha remove the winner puts statement

Reply:
Humor gets under appreciated in coding comments.